cypress/support/pages/course/pageCourses.js

```diff
@@ -1039,7 +1049,7 @@ class Courses {
 	}
 
 	seeStartDateAndEndDatePickersAreDisabled() {
-		cy.get(Courses.#courseStartDatePicker).should("be.disabled");
+		cy.get(Courses.#courseEndDatePicker).should("be.disabled");
```

Comment:
These twso lines are identical, shouldn't it be #courseStartDatePicker in the first line?

Reply:
fixed!